FrameworkSystem/scripts/dirac-populate-component-db.py

```diff
@@ -43,15 +44,25 @@ def setExcludedHosts( value ):
 
 componentType = ''
 
+result = getProxyInfo()
+if not result[ 'OK' ]:
+  gLogger.error( result[ 'Message' ] )
+  DIRACexit( -1 )
+
+# If the user is not a ServiceAdministrator the script will not work
+if not 'ServiceAdministrator' in result['Value']['groupProperties']:
+  gLogger.error( 'ServiceAdministrator property missing' )
+  DIRACexit( -1 )
+
```

Comment:
This is a standard mistake in the scripts. The access rules must be imposed on the service and no point to check them in the scripts